scripts/setup_fvp_optee.sh

```diff
@@ -216,7 +218,8 @@ cat >> $DST_GEN_ROOTFS/filelist-tee.txt << EOF
 # OP-TEE device
 dir /lib/modules 755 0 0
 dir /lib/modules/$KERNEL_VERSION 755 0 0
-file /lib/modules/$KERNEL_VERSION/optee.ko $DST_OPTEE_LK/optee.ko 755 0 0
+file /lib/modules/$KERNEL_VERSION/optee.ko $DST_OPTEE_LK/core/optee.ko 755 0 0
+file /lib/modules/$KERNEL_VERSION/optee_armtz.ko $DST_OPTEE_LK/armtz/optee_armtz.ko 755 0 0
```

Comment:
I'm fine with changing the name, but should we be really picky, then we should probably name it something like "linux kernel driver", since you can have a linux driver in user space also. No matter what we change it to, what about using a shortened name? LD? LKD?